src/Altinn.Notifications/Authorization/IAuthorizationService.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Altinn.Notifications.Authorization;
+
+/// <summary>
+/// Describes the necessary functions of an authorization service that can perform
+/// notification recipient filtering based on authorization
+/// </summary>
+public interface IAuthorizationService
+{
+    /// <summary>
+    /// Describes a method that can create an authorization request to authorize a set of
+    /// users for access to a resource.
+    /// </summary>
+    /// <param name="orgRightHolders">The list organizations with associated right holders.</param>
```

Comment:
Think we need to be explicit somewhere in what the two ints are. I cannot find partyId and userId mentioned anywhere  at the moment.. We need a more explicit definition of this, so everyone can understand.